src/effective_couplings.hpp

```diff
@@ -49,6 +51,11 @@ std::complex<double> quadratic_interpolation(
 std::map<double,std::complex<double> > get_scalar_fermion_loop_data();
 std::map<double,std::complex<double> > get_pseudoscalar_fermion_loop_data();
 
+template<typename T>
```

Comment:
I think this function should be moved into `wrappers.hpp`.

Reply:
I was under the impression that `wrapper.hpp` should contain C++ versions of mathematica functions. Hence the name wrapper.

Reply:
You are right. 

However, I was wondering why this function was put into `src/effective_couplings.hpp `. To my understanding this function appears in many loop functions and not necessarily in effective couplings only. So, I thought there should be a better place. For example, we have `src/threshold_loop_functions.hpp` where loop functions are defined. (BTW, in `src/threshold_loop_functions.cpp` there is already this lambda function defined, but used only locally.)

Reply:
If you're using this function only locally, you may put it into an anonymous namespace in that module. What do you think?

Reply:
I think this was originally introduced by Dylan when he was working on effective couplings. I tweaked it but it stayed in `src/effective_couplings.hpp`.
Maybe we could have it in place? And replace the one in `src/threshold_loop_functions.cpp` with some common one? It can be in `wrapper.cpp`.

Reply:
For me that would be ok to define it in`wrappers.hpp`:
~~~
double Kallenlambda(double) noexcept;
~~~
In `wrappers.cpp` we have the corresponding definition.

However, I would keep `threshold_loop_functions.cpp` as it is and not include `wrappers.hpp` there, in order to keep inter-dependencies minimal and not include stuff that is not needed.